src/PhpSpec/CodeAnalysis/AccessInspector.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace PhpSpec\CodeAnalysis;
+
+use ReflectionMethod;
+use ReflectionProperty;
+
+class AccessInspector implements AccessInspectorInterface
```

Comment:
The Mage Caller doesn't allow magic __get, __set or __call methods to be used for determining if properties and methods are accessible.